<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGoods</class>
 <widget class="QMainWindow" name="AddGoods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>659</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Способ доставки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>375</y>
      <width>659</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить товар</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>308</y>
      <width>659</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Название картинки товара (*.jpg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>189</y>
      <width>659</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество товара</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>249</y>
      <width>659</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ссылка на справочную информацию</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="link_to_documentation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>279</y>
      <width>659</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>659</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Название товара</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dilivery">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>659</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>659</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_to_jpg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>339</y>
      <width>659</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_goods">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>659</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>659</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Цена</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_goods">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>659</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>659</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
